test/e2e/kwokctl/dryrun/main_test.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/14136b3c2da355bc846279af58c4696f26d123f9/.github/workflows/test.yaml#L252-L261

Add a step on ci to run this test, like `./hack/e2e-test.sh e2e/kwokctl/dryrun`

Reply:
done